### ISSUE TITLE
[Tablets] Support stream_io_throughput_mb_per_sec for tablets streaming

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylladb#19861]
Unlike vnodes-based streaming and RBNO, tablets streaming does not currently support throttling by `stream_io_throughput_mb_per_sec`.

**Comment by bhalevy** [comment]
@michoecho:
> That's a seastar-level knob. How is it possible not to support it? (edited) 

> It throttles IO for the streaming group on the IO scheduler level.

@bhalevy:
> If that's the case, we'll need to adjust the dtests that exercise it for tablets.
> They currently depend on streamiong/rbno-specific log messages that aren't printed with tablets.

@denesb:
> I don't understand how that can be a seastar level knob. Seastar doesn't know anything about ScyllaDB scheduling groups? You mean it directly tweaks things related to the streaming IO group?

@michoecho:
> Seastar has an API for setting bandwidth limits on scheduling groups, and Scylla just calls that API on the streaming group whenever stream_io_throughput_mb_per_sec is updated (edited)

**Comment by denesb** [comment]
@bhalevy do I understand correctly that this issue is just about adjusting tests?

**Comment by bhalevy** [comment]
> [@bhalevy](https://github.com/bhalevy) do I understand correctly that this issue is just about adjusting tests?

Yes.  Initially I thought we need to fix this on the scylla side, but we came to a conclusion throttling is supposed to work, so we need to adjust the tests for that.
Maybe a good opportunity to migrate the relevant dtests to test.py
